### PR TITLE
refact: update all VOs to new format

### PR DESCRIPTION
**Pull request by polillomm**

### CODE REVIEW

File: src/domain/valueObject/networkPort.go [hunk 1]
```diff
@@ -9,16 +9,16 @@ import (
 
 type NetworkPort uint
```
Comment (ntorga) [inline src/domain/valueObject/networkPort.go:10]: uint16

File: src/domain/valueObject/marketplaceItemType.go [hunk 2]
```diff
@@ -4,40 +4,34 @@ import (
...
 )
 
 type MarketplaceItemType string
 
-var ValidMarketplaceItemTypes = []string{
+var validMarketplaceItemTypes = []string{
 	"app",
 	"framework",
 	"stack",
 }
```
Comment (ntorga) [inline src/domain/valueObject/marketplaceItemType.go:17]: Eliminate blank lines.

**Review by ntorga — CHANGES_REQUESTED** [review]

File: src/domain/valueObject/networkPort.go (hunk 1 above)
Comment (polillomm) [reply src/domain/valueObject/networkPort.go:10]: Done!

File: src/domain/valueObject/marketplaceItemType.go (hunk 2 above)
Comment (polillomm) [reply src/domain/valueObject/marketplaceItemType.go:17]: Done!

**Pull request merged**